### PR TITLE
Feature/contact selection ui

### PR DESCRIPTION
**Pull request by Schwarzglut**
The skeleton for the contact screen
Displaying hard-coded contacts through a contact gameobject prefab that is spawned to screen
To add: 
-grab contact details from JSON
-Positioning of contacts taken from screen height or scroll rect
-remove hard-coding
-trigger message ui from contact press

### CODE REVIEW

File: Assets/Prefabs/Contact.prefab [hunk 1]
```diff
@@ -0,0 +1,330 @@
+%YAML 1.1
```
Comment (Schwarzglut) [inline Assets/Prefabs/Contact.prefab]: Oh absolutely, I'll fix that quickly!

Comment (Schwarzglut) [inline Assets/Prefabs/Contact.prefab]: Would you find if I looked at this tomorrow?

Comment (imann24) [inline Assets/Prefabs/Contact.prefab]: I'm sorry to be such a pain here, but can you also bump the contact screen up in the z-sort order? It's rendering behind the home screen

Comment (Schwarzglut) [inline Assets/Prefabs/Contact.prefab]: No problem at all. Hopefully that did it.

**Pull request merged**